src/encdatavault_fmt_plug.c

```diff
@@ -185,19 +210,25 @@ static int valid(char *ciphertext, struct fmt_main *self)
 	ctcopy += FORMAT_TAG_LENGTH;
 	if ((p = strtokm(ctcopy, "$")) == NULL) // version
 		goto err;
-	if (strcmp(p, "1") && strcmp(p, "3")) {
+	if (!isdec(p))
+		goto err;
+	int version = atoi(p);
+
+	if (version != 3 && version != 1 && version != 0x13 && version != 0x11) {
 		static int warned;
+
 		if (!self_test_running && !warned++)
-			fprintf(stderr, "%s: Warning: version %d not supported, not loading such hashes!\n", self->params.label, atoi(p));
+			fprintf(stderr, "%s: Warning: version %d not supported, not loading such hashes!\n", self->params.label,
+			        atoi(p));
```

Comment:
Drop the empty lines (I think they hurt clarity), unwrap the line (OK to go to 131 chars), replace the second `atoi(p)` with `version`.

Reply:
The empty line was done by using the format code commands in `CONTRIBUTING.md`.

Reply:
`indent`